public-front/docker/app/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM composer AS composer
 
 COPY public-front /app
-
+RUN apk add icu-dev
+RUN docker-php-ext-install -j$(nproc) \
```

Comment:
This is an interesting one: should we just ignore the reqs at the point where we do the install and rely on the proper container to fail if *it* is missing the reqs; or enforce the reqs on the composer image. In the past I have explicitly run composer on the container at the end of the chain, rather than doing it in a separate container at the start (as we have here); but that means installing composer on the container and potentially removing it afterwards. On the plus side, the composer install can enforce the PHP version and ensure that all required PHP modules are available on the container: a kind of install-time dependency check. I don't think we have that with the current arrangement.

@williamfalconeruk What's your preference?

Reply:
for this project, probably not worth spending time on this here, but definitely something we could look at on the lpa live project, to see the benefits of it.